src/main/java/com/g4s8/ghman/web/TkGitHubAuthorization.java

```diff
@@ -60,6 +58,12 @@
      */
     private final Func<org.takes.Request, String> auth;
 
+    /**
+     * Logger.
+     */
+    @SuppressWarnings("PMD.LoggerIsNotStaticFinal")
+    private final Logger logger;
```

Comment:
@victornoel logger -> log ([Don't Create Objects That End With -ER](https://www.yegor256.com/2015/03/09/objects-end-with-er.html))